test/e2e/create_cluster_test.go

```diff
@@ -1374,7 +1374,12 @@ func TestCreateClusterCustomConfig(t *testing.T) {
 
 		// ensure image registry component is disabled
 		e2eutil.EnsureImageRegistryCapabilityDisabled(ctx, t, g, mgtClient, hostedCluster)
-		e2eutil.EnsureKubeAPIDNSName(t, ctx, mgtClient, hostedCluster)
+
+		t.Run("KubeAPIDNSNameTests", func(t *testing.T) {
+			// These tests will run in series within this subtest
+			e2eutil.EnsureKubeAPIDNSName(t, ctx, mgtClient, hostedCluster)
```

Comment:
what's EnsureKubeAPIDNSName convering that is not covered by EnsureKubeAPIDNSNameCustomCert? can we document them?

Reply:
Based on the discussion from slack, let's remove the `EnsureKubeAPIDNSName` test.